src/pybind/mgr/dashboard/module.py

```diff
@@ -184,6 +184,14 @@ def notify(self, notify_type, notify_val):
                     self.log_buffer.appendleft(notify_val)
         elif notify_type == "pg_summary":
             self.update_pool_stats()
+
+            #pg_summary = self.get("pg_summary")
```

Comment:
this change is not related to the noop stub of `os._exit()`. can we drop it?

Reply:
oops!  done.